presto-main/src/test/java/io/prestosql/server/security/TestUserMapping.java

```diff
@@ -70,29 +70,29 @@ public void testSimplePatternRule()
     public void testReplacePatternRule()
             throws Exception
     {
-        UserMapping userMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "$1 ^ $1", true)));
+        UserMapping userMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "$1 ^ $1", true, UserMapping.Case.KEEP)));
         assertEquals(userMapping.mapUser("test@example.com"), "test ^ test");
         assertThrows(UserMappingException.class, () -> userMapping.mapUser("no at sign"));
 
-        UserMapping emptyMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "  ", true)));
+        UserMapping emptyMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "  ", true, UserMapping.Case.KEEP)));
         assertThrows(UserMappingException.class, () -> emptyMapping.mapUser("test@example.com"));
     }
 
     @Test
     public void testNotAllowedRule()
     {
-        UserMapping userMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "$1", false)));
+        UserMapping userMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "$1", false, UserMapping.Case.KEEP)));
         assertThrows(UserMappingException.class, () -> userMapping.mapUser("test@example.com"));
 
-        UserMapping emptyMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "", false)));
+        UserMapping emptyMapping = new UserMapping(ImmutableList.of(new Rule("(.*?)@.*", "", false, UserMapping.Case.KEEP)));
         assertThrows(UserMappingException.class, () -> emptyMapping.mapUser("test@example.com"));
     }
 
     @Test
     public void testMultipleRule()
             throws Exception
     {
-        UserMapping userMapping = new UserMapping(ImmutableList.of(new Rule("test@example.com", "", false), new Rule("(.*?)@example.com")));
+        UserMapping userMapping = new UserMapping(ImmutableList.of(new Rule("test@example.com", "", false, UserMapping.Case.KEEP), new Rule("(.*?)@example.com")));
```

Comment:
Static import

Reply:
can't somebody configure checkstyle / sonar / intellij rules for that, once and for all? This is what intellij does by default after typing KEEP

Reply:
It is hard to have a rule for a class that you have just added. Also I would prefer to not have such rule for `Case` class as in this context it is clear what it is, but in other it might not be.